configure-client.html

```diff
@@ -0,0 +1,368 @@
+---
+layout: default
+title: Configuring OMERO Clients
+menu-id: l1-workflow2
+pdfs:
+    - configure-client.pdf
+description: How to configure the OMERO.insight client, adding third-party jars, increasing allocated memory.
+version:
+    - yes
+redirect_from:
+  - /client-memory.html
+---
+<div class="page-navigation">
+<h2>Contents</h2>
+<p>
+<a href="#introduction">Introduction</a><br />
+<a href="#jar">Adding third-party .jars to OMERO.insight</a><br />
+<a href="#jarmacosx" class="indent1">OMERO.insight on OSX</a><br />
+<a href="#jarwindows" class="indent1">OMERO.insight on Windows</a><br />
+<a href="#jarlinux" class="indent1">OMERO.insight on Linux/Unix</a><br />
+<a href="#increase">Increasing OMERO.insight memory allocation</a><br />
+<a href="#increasemacosx" class="indent1">OMERO.insight on OSX</a><br />
+<a href="#increasewindows" class="indent1">OMERO.insight on Windows</a><br />
+<a href="#increaselinux" class="indent1">OMERO.insight on Linux/Unix</a><br />
+</p>
+</div><!-- End Page Navigation -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="introduction"></a>Introduction</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p>The OMERO.insight client is a Java based desktop application that runs on Windows, Mac and Unix/Linux. There are a number of ways the client can be configured to improve or extend its functionality.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">Configuring your client application depends on the operating system and the client used. Also note that this guide does not cover OMERO.web, which uses a different, server-based, configuration set-up.</p>
```

Comment:
I don't think you really need 'set-up' at the end of this sentence - 'configuration' covers it.